index.html

```diff
@@ -0,0 +1 @@
+<h1>Hello world!</h1>
```

Comment:
It is working, but common htmp page should contain not only <h1> tag. Here is A Simple HTML Document example (https://www.w3schools.com/html/html_intro.asp)